src/components/File.jsx

```diff
@@ -143,6 +145,7 @@ class File extends Component {
     const { filename, extension } = splitFilename(attributes)
     return (
       <div className={classes}>
+        { attributes.links && <Preview thumbnail={`${cozy.client._url}${attributes.links.small}`} /> }
```

Comment:
Or may the `<Preview />` component be connected to load the required information?